### PR TITLE
feat(recipient): Remove v from recipients/contacts

### PR DESCRIPTION
**Pull request by finnian0826**
### Description

Remove the V icon from recipients/contacts who are on Valora.

Reason for the change: [Linear issue](https://linear.app/valora/issue/ACT-782/remove-the-v-icon-on-contacts-recipients), [Notion discussion](https://www.notion.so/valora-inc/Recipient-screen-Valora-V-behavior-658afd84498345d38027fc43df67397d)

### Test plan

Before change:
v_icon_before

After change:
v_icon_after

### CODE REVIEW

File: src/recipients/RecipientItem.tsx [hunk 1]
```diff
@@ -33,13 +32,6 @@ function RecipientItem({ recipient, onSelectRecipient, loading }: Props) {
           </Text>
           {recipient.name && <Text style={styles.phone}>{getDisplayDetail(recipient)}</Text>}
         </View>
-        <View style={styles.rightIconContainer}>
-          {loading ? (
```
Comment (satish-ravi) [inline src/recipients/RecipientItem.tsx:37]: I think we still need the loading icon here. We want to show it when looking up whether a phone number is connected to an address. Can you confirm this by typing a phone number on the search box and tapping it?

Comment (cajubelt) [reply src/recipients/RecipientItem.tsx:37]: @satish-ravi are you suggesting just using the loading prop to show a spinner that turns into a blank space once the lookup is complete? why do we need that?

Comment (cajubelt) [reply src/recipients/RecipientItem.tsx:37]: oh ic, a bottom sheet comes up once the lookup completes successfully

Comment (cajubelt) [reply src/recipients/RecipientItem.tsx:37]: and an error banner appears if it fails

Comment (satish-ravi) [reply src/recipients/RecipientItem.tsx:37]: yeah, I think it also navigates to an invite screen if its a phone number not connected to Valora

**Comment by jeanregisser** [comment]
Thanks! 👍 

Even though it's a small change, could you add some info to help folks not familiar with this (me? 😄) why we're making this change? Could be a linear task, slack convo, or just some text in the PR. 🙏

**Review by cajubelt — APPROVED** [review]

**Pull request merged**